packages/grafana-data/src/types/dataFrame.ts

```diff
@@ -112,6 +112,9 @@ export interface FieldConfig<TOptions = any> {
 
   // Calculate min max per field
   fieldMinMax?: boolean;
+
+  // Enable time compare
+  timeCompare?: boolean;
```

Comment:
Don't think this should be field config. It feels to me like time comparison should be enabled either per-query or per-panel. Not per field.